src/core/account/plugin-api.js

```diff
@@ -60,13 +60,16 @@ export class CurrencyConfig extends Bridgeable<EdgeCurrencyConfig> {
     )
   }
 
-  async importKey(userInput: string): Promise<JsonObject> {
+  async importKey(
+    userInput: string,
+    opts?: { keyOptions: JsonObject }
```

Comment:
We prefer the `opts: {...}  = {}` where possible, rather than the `?.`. Generally, `??` and `?.` should be avoided in favor of default parameters / default destructuring where possible, since the latter are more explicit, and reduce the number of `undefined` values floating around the program as soon as possible.